crates/solidity/outputs/typescript/lib/.github/workflows/lint.yml

```diff
@@ -0,0 +1,41 @@
+name: Lint
```

Comment:
I suggest removing that file, since it has no use for us moving forward.

Reply:
done